.travis.yml

```diff
@@ -1,7 +1,52 @@
 language: objective-c
-osx_image: xcode7
+osx_image: xcode7.2
+
+sudo: false
+branches:
+  only:
```

Comment:
This here makes sure we only run the “push” job for commits to the `master` and any `release-`. PRs still run the PR job. Limiting the push job to only these branches makes sure we don’t double the matrix for PRs. The PR branch is still built with latests master merged in, so all is fine.